client/src/components/settings/ProfileInformationForm.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <v-form>
+    <v-text-field
+      v-model="email"
+      :counter="10"
+      disabled
+      label="E-mail"
+    />
+
+    <v-text-field
+      v-model="state.firstName"
+      label="First Name"
+      required
+      :rules="nameRules"
+    />
+
+    <v-text-field
+      v-model="state.lastName"
+      label="Last Name"
+      required
+      :rules="nameRules"
+    />
+
+    <v-text-field
+      v-model="state.phoneNumber"
+      label="Phone Number"
+      :rules="phoneNumberRules"
+    />
+
+    <v-btn
+      class="me-4"
+      type="submit"
+      @click="putProfileSettings"
+    >
```

Comment:
```suggestion
      @click.stop.prevent="putProfileSettings"
```

This would prevent default behavior which is reloading the page again from happening.

Reply:
Also, can we show an alert informs the user that the profile has been edited? use v-alert.